conu.spec

```diff
@@ -0,0 +1,43 @@
+%global framework_name conu
```

Comment:
I would prefer to move spec into directory rpm.

Reply:
Then we'll have same issue  as in MTF, `tito` will not work well and `COPR` will have troubles with using `scm` builds

Reply:
tito? please don't